acceptance/acceptance_test.go

```diff
@@ -846,7 +848,7 @@ func testAcceptance(
 								assertions.NewOutputAssertionManager(t, output).ReportsSuccessfulImageBuild(repoName)
 
 								assertOutput := assertions.NewLifecycleOutputAssertionManager(t, output)
-								assertOutput.IncludesLifecycleImageTag(lifecycle.Image())
+								assertOutput.IncludesLifecycleImageTag(origLifecycle)
```

Comment:
nit: maybe we should call this `IncludesTagOrEphemeralLifecycle` (or something) to indicate the fallback